### PR TITLE
Bump aiohttp to 3.7.1

### PR DESCRIPTION
**Pull request by asvetlov**

### CODE REVIEW

File: neuromation/api/storage.py [hunk 1]
```diff
@@ -121,7 +121,10 @@ def _uri_to_path(self, uri: URL, *, normalized: bool = False) -> str:
 
     def _set_time_diff(self, request_time: float, resp: aiohttp.ClientResponse) -> None:
         response_time = time.time()
-        server_timetuple = parsedate(resp.headers.get("Date"))
```
Comment (serhiy-storchaka) [inline neuromation/api/storage.py:124]: See also https://github.com/python/typeshed/issues/4713.

Although the behavior of `parsedate(None)` is not explicitly documented and therefore your code is more future proof.

File: tests/cli/test_main.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def test_steal_config(
     monkeypatch: Any,
 ) -> None:
     patched_jobs_counts.return_value = {}
-    folder = shutil.move(nmrc_path, tmp_path / "orig-cfg")  # type: ignore
+    folder = shutil.move(str(nmrc_path), str(tmp_path / "orig-cfg"))
```
Comment (serhiy-storchaka) [inline tests/cli/test_main.py:24]: Why do you need to convert paths to str? Does not `shutil.move()` support the path protocol?

File: neuromation/api/storage.py (hunk 1 above)
Comment (asvetlov) [reply neuromation/api/storage.py:124]: this was my thought

File: tests/cli/test_main.py (hunk 2 above)
Comment (asvetlov) [reply tests/cli/test_main.py:24]: mypy had blamed that `move` requires `str`.
Perhaps it is a typeshed spike.
I did not investigate but quickly converted to the required type.
It is the test only, I wanted to make it pass quickly.

Comment (hauntsaninja) [reply tests/cli/test_main.py:24]: I think typeshed's annotations are working around https://bugs.python.org/issue32689 (fixed in 3.9, reflected in typeshed).

**Pull request merged**